### PR TITLE
add civic dataset description

### PR DESCRIPTION
**Pull request by malachig**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 71baefc17e52607c98c7053b03f118edbfbdc6d5
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e2bd6dd5-35bd-45f1-b95c-1d459deea576%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by chueatwork** [comment]
@malachig this is just failing code build because we don't have a resource listed, easy to remedy once we set up an S3 bucket!

**Comment by chueatwork** [comment]
@malachig if you can add the S3 bucket ARN (civic-aws-opendata) and region and then rebase, we can merge this PR! Thanks!

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: a66ad486cc2c18f973ad9adfecb34ad6e5c30fde
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=8cf21b3b-cb6b-44e1-93ed-17210282f872%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachig** [comment]
@chueatwork okay, I think this is done now.

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 8b76a6c940523b5efe33a39b45b604c3b96d40bf
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=bd8085f0-7cc4-4cc0-990c-e7d90744f5e3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/civic.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - genetic
+  - genomic
+  - life sciences
+License: CC0 (https://creativecommons.org/publicdomain/zero/1.0/)
+Resources:
+  - Description: Monthly CIViC data dumps
+    ARN: arn:aws:s3:::civic-aws-opendata
+    Region: us-west-2
+    Type: S3 Bucket
+    Explore:
```
Comment (zflamig) [inline datasets/civic.yaml:17]: Delete this line if you aren't using it

File: datasets/civic.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+DataAtWork:
+  Tutorials:
```
Comment (zflamig) [inline datasets/civic.yaml:19]: Delete these unused Tutorials and Tools & Applications blocks

**Comment by chueatwork** [comment]
One more rebase and we're RTG!

**Comment by malachig** [comment]
Okay, done.

**Pull request merged**